src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -116,7 +119,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
   const goToChatScreen = (channel: ChannelList, from) => {
     setChat({
       ...chat,
-      selectedChannel: channel
+      selectedChannel: channel as unknown as null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The line `selectedChannel: channel as unknown as null` seems incorrect. The `as unknown as null` part will cast `channel` to `null`, which doesn't seem to be the intended behavior based on the old hunk. If you want to set `selectedChannel` to `channel`, you should just assign it directly without casting.
```diff
-      selectedChannel: channel as unknown as null
+      selectedChannel: channel
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->